### PR TITLE
added unit test for processOperation

### PR DESCRIPTION
**Pull request by sadiq1971**

### CODE REVIEW

File: test/unit/chain/stateTransition/block/operations/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('should fail to process operations - outstanding deposits not processed up to the maximum', function () {
+    const state  = generateState();
+    const body = generateEmptyBlock().body;
+    body.deposits.push(generateDeposit());
+    try {
+      processOperations(state, body);
```
Comment (wemeetagain) [inline test/unit/chain/stateTransition/block/operations/index.test.ts:65]: Maybe use expect(...).to.throw() here and elsewhere
eg:
```typescript
expect(() => processOperations(state, body)).to.throw();
```

**Review by wemeetagain — CHANGES_REQUESTED** [review]
lgtm! minor comment

File: test/unit/chain/stateTransition/block/operations/index.test.ts (hunk 1 above)
Comment (sadiq1971) [reply test/unit/chain/stateTransition/block/operations/index.test.ts:65]: ok

**Review by wemeetagain — APPROVED** [review]

**Review by GregTheGreek — APPROVED** [review]

**Comment by GregTheGreek** [comment]
CLA is stuck...

**Comment by GregTheGreek** [comment]
Closed to re-trigger CLA-Assistant

**Pull request merged**